packages/flutter/lib/src/widgets/animated_switcher.dart

```diff
@@ -63,51 +68,55 @@ typedef Widget AnimatedSwitcherLayoutBuilder(List<Widget> children);
 /// different parameters, then [AnimatedSwitcher] will *not* do a
 /// transition between them, since as far as the framework is concerned, they
 /// are the same widget, and the existing widget can be updated with the new
-/// parameters. If you wish to force the transition to occur, set a [Key]
-/// (typically a [ValueKey] taking any widget data that would change the visual
-/// appearance of the widget) on each child widget that you wish to be
-/// considered unique.
+/// parameters. To force the transition to occur, set a [Key] (typically a
+/// [ValueKey] taking any widget data that would change the visual appearance
+/// of the widget) on each child widget that you wish to be considered unique.
 ///
 /// ## Sample code
 ///
 /// ```dart
 /// class ClickCounter extends StatefulWidget {
 ///   const ClickCounter({Key key}) : super(key: key);
-///
+/// 
 ///   @override
 ///   _ClickCounterState createState() => new _ClickCounterState();
 /// }
 ///
 /// class _ClickCounterState extends State<ClickCounter> {
 ///   int _count = 0;
-///
+/// 
 ///   @override
 ///   Widget build(BuildContext context) {
 ///     return new Material(
-///       child: Column(
-///         mainAxisAlignment: MainAxisAlignment.center,
-///         children: <Widget>[
-///           new AnimatedSwitcher(
-///             duration: const Duration(milliseconds: 200),
-///             transitionBuilder: (Widget child, Animation<double> animation) {
-///               return new ScaleTransition(child: child, scale: animation);
-///             },
-///             child: new Text(
-///               '$_count',
-///               // Must have this key to build a unique widget when _count changes.
-///               key: new ValueKey<int>(_count),
-///               textScaleFactor: 3.0,
+///       child: new Directionality(
```

Comment:
see earlier discussion

Reply:
I changed the example to just use a MaterialApp instead of Directionality.